data/avro-schema/src/main/resources/avro/net/corda/data/messaging/mediator/MediatorReplayOutputEvents.avsc

```diff
@@ -0,0 +1,21 @@
+{
+  "type": "record",
+  "name": "MediatorReplayOutputEvents",
+  "namespace": "net.corda.data.messaging.mediator",
+  "doc": "Record the output records for a given input record, to be replayed by the mediator if necessary.",
+  "fields": [
+    {
+      "name": "key",
```

Comment:
Slight nit: I'd rename this field to `hash` or `checksum` or something, as right now `MediatorReplayOutputEvent.key` and `MediatorReplayOutputEvents.key` mean different things.